bin/check_samplesheet.py

```diff
@@ -100,7 +102,8 @@
     if len(outLines) != 1:
         print "{}: Only a single-line can be specified in samplesheet without barcode information!".format(ERROR_STR)
         sys.exit(1)
-    outLines[0][2] = 'barcode%s' % ('1'.zfill(2))
+    ## USE SAMPLE NAME AS BARCODE WHEN NOT DEMULTIPLEXING
+    outLines[0][2] = sample
```

Comment:
The `sample` variable doesnt exist here?

Reply:
It does from the previous loop but maybe safer to just use list indexing to allocate that entry?